scenario/context.py

```diff
@@ -353,15 +353,15 @@ def _coalesce_event(event: Union[str, Event]) -> Event:
             )
         return event
 
-    def manager(
+    def event(
```

Comment:
Could we add a return type hint?

Specifically, does this hold?

```py
ctx = scenario.Context(...)
with ctx.event(...) as sub:
    assert sub is ctx
```

Or is `sub` different, yet still same type...

Or is `sub` something of a different kind entirely?